frmMain.frm

```diff
@@ -234,24 +233,49 @@ Private Sub Image1_Click()
 End
 End Sub
 
-Private Sub Image1_MouseDown(Button As Integer, Shift As Integer, x As Single, Y As Single)
+Private Sub Image1_MouseDown(Button As Integer, Shift As Integer, X As Single, Y As Single)
     Image1.Picture = LoadPicture(App.Path & "\Graficos\AU_Salir_A.jpg")
 End Sub
 
-Private Sub Image1_MouseMove(Button As Integer, Shift As Integer, x As Single, Y As Single)
+Private Sub Image1_MouseMove(Button As Integer, Shift As Integer, X As Single, Y As Single)
     Image1.Picture = LoadPicture(App.Path & "\Graficos\AU_Salir_I.jpg")
 End Sub
+
+Public Function GetIEVersion()
+    Dim FileSystemObject As New FileSystemObject
+    Dim Version As String
+    
+    Version = FileSystemObject.GetFileVersion("c:\windows\system32\ieframe.dll")
+    GetIEVersion = Version
+End Function
+
+Public Function CheckIfIEVersionIsCompatible()
+    Dim IEVersion As String
+    Dim IEVersionArray() As String
+
+    IEVersion = GetIEVersion
+    IEVersionArray() = Split(IEVersion, ".")
+
+    If CInt(IEVersionArray(0)) < 10 Then
+        MsgBox ("Tu version de Internet Explorer " & IEVersionArray(0) & " no es compatible. Necesitas Internet Explorer 11 o una version superior para poder utilizar el actualizador.")
+        MsgBox ("Si utilizas Windows XP o no podes actualizar Internet Explorer a la version 11 o superior, por favor lee el siguiente post para la instalacion del juego: https://www.reddit.com/r/argentumonlineoficial/comments/9ywcvb/instalacion_windows_xp_importante/")

```

Comment:
Can they click on that link inside a MsgBox?

Reply:
No, I didn't know how to do it so I made it simple.

Reply:
That's fine, that will work for now